gnomad/resources/import_resources.py

```diff
@@ -4,15 +4,15 @@
 import itertools
 import textwrap
 from inspect import getmembers
-from typing import Dict, Tuple, Optional
+from typing import Dict, Optional, Tuple
```

Comment:
so it IS convention to use alphabetical imports!

Reply:
Yup!!